cpp/CMakeLists.txt

```diff
@@ -109,9 +95,13 @@ include(cmake/Modules/ConfigureCUDA.cmake)
 # - dependencies ----------------------------------------------------------------------------------
 
 # find gdal
+# TODO: How do we add this to rapids-export? I don't think we want to
+# rapids-cpm-find this because we're not looking to download and build a local
+# copy, so should we be using one of the rapids_export_* functions to manually
+# add it to the list?
```

Comment:
You want to use `rapids_find_package` which behaves like `find_package` but with depdency tracking.

https://docs.rapids.ai/api/rapids-cmake/stable/command/rapids_find_package.html

Reply:
Ah right OK so `rapids_cpm_find` will perform dependency tracking _and_ use CPM to download a package if it's not found locally, whereas `rapids_find_package` will perform dependency tracking but _will not_ download and build missing packages? In this case we definitely don't want to build GDAL locally.

Reply:
correct!